core/new-gui/src/app/workspace/component/operator-panel/operator-label/operator-label.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-card class="texera-operator-label mat-elevation-z1" [attr.id]="operatorLabelID">
```

Comment:
F2FQ: What is [attr.id]'s brackets mean? Angular Tutorial said it meant input into the child component. But here mat-card is not a child.

Reply:
[attr.id] is set the DOM element's ID using a variable in the class.
`id=something` will interpret `something` as a string
`[attr.id]=something` will interpret `something` as a public variable in the component class. It's Angular's syntax.